language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/tests/helpers/DummyLedgerClient.scala

```diff
@@ -35,6 +35,12 @@ class DummyLedgerClient(
         filter: TransactionFilter,
         verbose: Boolean): Flowable[GetActiveContractsResponse] =
       activeContractSet
+
+    override def getActiveContracts(
+        filter: TransactionFilter,
+        verbose: Boolean,
+        accessToken: String): Flowable[GetActiveContractsResponse] =
+      ???
```

Comment:
Please add a comment that explains shortly why this can/should be `???`.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3653/commits/aa780acad3c08acb02bddd6a4e713678fe556cc3